src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -4,9 +4,9 @@
 import java.time.format.DateTimeFormatter;
 
 public class SalaryInfo {
-    static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
-    static final int DATEINDEX = 0;
-    static final int NAMEINDEX = 1;
+    static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
let's not forget about access modifiers. and return `FORMATTER` to its previous state

Reply:
I'm sorry, but I don't understand what type of changes should I make for access modifiers. Could you explain, please?